txnkv/transaction/pipelined_flush.go

```diff
@@ -338,6 +338,23 @@ func (c *twoPhaseCommitter) commitFlushedMutations(bo *retry.Backoffer) error {
 		zap.Uint64("startTS", c.startTS),
 		zap.Uint64("commitTS", commitTS),
 	)
+	broadcastToAllStores(
```

Comment:
The function call will wait for the broadcast done in current goroutine, though this won't affect too much for p-dml, shall we put it in an individual goroutine instead of blocking the statement from returning success to client?

Reply:
Good idea